ext/spl/spl_iterators.c

```diff
@@ -650,6 +665,7 @@ static void spl_recursive_it_it_construct(INTERNAL_FUNCTION_PARAMETERS, zend_cla
 	intern->iterators[0].getchildren = NULL;
 
 	if (EG(exception)) {
+		// TODO: use spl_RecursiveIteratorIterator_free_iterators
 		zend_object_iterator *sub_iter;
```

Comment:
TODO Comment?

Reply:
Yes, but I thought it's best to only the refactor on master. So I left the TODO for when I merge up or for a follow-up

Reply:
ACK, makes sense.